idaes/generic_models/unit_models/column_models/solvent_column.py

```diff
@@ -622,38 +624,37 @@ def rule_heat_transfer_coeff_Ack(blk, t, x):
             rule=rule_heat_transfer_coeff_Ack,
             doc='Vap-Liq heat transfer coefficient corrected by Ackmann factor')
 
-        # Heat flux  - vapor side [J/s.m]
+        # Convective interphase heat transfer [J/s.m]
         @self.Constraint(self.flowsheet().time,
                          self.vapor_phase.length_domain,
-                         doc="heat transfer - vapor side ")
-        def vapor_phase_volumetric_heat_flux(blk, t, x):
+                         doc="Convective interphase heat transfer")
+        def vapor_phase_convective_heat_flow(blk, t, x):
             if x == self.vapor_phase.length_domain.first():
-                return blk.heat_flux_vap[t, x] == 0
+                return blk.heat_flow_vap[t, x] == 0
             else:
                 zb = self.vapor_phase.length_domain.at(value(self.zi[x]) - 1)
-                return blk.heat_flux_vap[t, x] == blk.h_v_Ack[t, x] * \
-                    (blk.liquid_phase.properties[t, zb].temperature -
-                      blk.vapor_phase.properties[t, x].temperature)
+                return blk.heat_flow_vap[t, x] == blk.h_v_Ack[t, x] * \
```

Comment:
Why did you update this to use the average temperature between the two points? I believe this model is written explicitly using a finite element approach assuming that the state in each cell is equal to the value of the finite element. I.e. I believe the original form was correct.

Reply:
I agree with this as well. I have changed it to the original equation.